stock_putaway_product/models/product_putaway.py

```diff
@@ -59,7 +59,7 @@ def get_default_inventory_id(self):
 
     putaway_id = fields.Many2one(
         comodel_name='product.putaway',
-        sting='Put Away Method',
+        string='Put Away Method',
```

Comment:
However, `sting` sounds better! :rofl: 
But joking apart, it's strange no error has been raised during CI builds.